docs/intro/overview.txt

```diff
@@ -3,8 +3,8 @@ Django at a glance
 ==================
 
 Because Django was developed in a fast-paced newsroom environment, it was
-designed to make common Web-development tasks fast and easy. Here's an informal
-overview of how to write a database-driven Web app with Django.
+designed to make common web-development tasks fast and easy. Here's an informal
```

Comment:
Can we chop also a hyphen?
```suggestion
designed to make common web development tasks fast and easy. Here's an informal
```

Reply:
Hyphens do not seem to be popular in `en-us`. AP say "generally we do not hypenate when using a prefix with a word starting with a consonant". Let us go with your suggestion as hopefully we can agree webdevelopment is not a word.

Reply:
> ... as hopefully we can agree webdevelopment is not a word.

Yes it's not a word, at least for now :wink: